exchanges/okx/okx_wrapper.go

```diff
@@ -2573,25 +2573,23 @@ func (ok *Okx) GetFuturesPositionOrders(ctx context.Context, req *futures.Positi
 	if err != nil {
 		return nil, err
 	}
+	contractsMap := make(map[currency.Pair]*futures.Contract)
+	for i := range contracts {
+		contractsMap[contracts[i].Name] = &contracts[i]
+	}
 	for i := range req.Pairs {
 		fPair, err := ok.FormatExchangeCurrency(req.Pairs[i], req.Asset)
 		if err != nil {
 			return nil, err
 		}
 		instrumentType := GetInstrumentTypeFromAssetItem(req.Asset)
 
-		multiplier := 1.0
-		var contractSettlementType futures.ContractSettlementType
-		if req.Asset.IsFutures() {
-			for j := range contracts {
-				if !contracts[j].Name.Equal(fPair) {
-					continue
-				}
-				multiplier = contracts[j].Multiplier
-				contractSettlementType = contracts[j].SettlementType
-				break
-			}
+		contract, exist := contractsMap[fPair]
+		if !exist {
```

Comment:
🐒 Nitpick:
* Should be `exists`, but `ok` is more idiomatic for go
```suggestion
		contract, ok := contractsMap[fPair]
		if !ok {
```

Tiny thing, can ignore.